rollbar-cli/ChangeLog.md

```diff
@@ -1,3 +1,13 @@
 # Changelog for rollbar-cli
 
+All notable changes to this project will be documented in this file.
+
+## [1.1.0] - 2024-05-28
+
+### Changed
+- Added support for GHC 9.4.7
+
+### Removed
+- Support for GHC 8.10.2 (in favor of GHC 8.10.7)
```

Comment:
@DavidMazarro same here; I believe we could omit this entry from the CHANGELOG because we are only transitioning between patch versions.